Library/Homebrew/cmd/audit.rb

```diff
@@ -121,10 +121,15 @@ def include?(s)
     @text.include? s
   end
 
-  def line_number(regex)
-    index = @lines.index { |line| line =~ regex }
+  def line_number(regex, skip = 0)
+    index = @lines.drop(skip).index { |line| line =~ regex }
```

Comment:
I add offset back in later which is why the code currently works, but absolute line number is probably better.